README.md

```diff
@@ -327,6 +327,10 @@ Unacceptable input example:
 /usr2/username/bin:/usr/local/bin:/usr/bin:.
 ```
 
+#### `Stdlib::Mac'
```

Comment:
backticks on both ends :)

Also, would you mind moving it up to _after_ Stdlib::Httpurl? you know, the alphabet and all.